sdk/storage/storage-queue/src/Pipeline.ts

```diff
@@ -139,45 +128,40 @@ export class Pipeline {
  * Option interface for newPipeline() method.
  *
  * @export
- * @interface NewPipelineOptions
+ * @interface StoragePipelineOptions
  */
-export interface NewPipelineOptions {
-  proxy?: ProxySettings | string;
+export interface StoragePipelineOptions {
+  /**
+   * Options to configure a proxy for outgoing requests.
+   */
+  proxyOptions?: ProxyOptions;
   /**
    * Telemetry configures the built-in telemetry policy behavior.
```

Comment:
Probably should update this doc comment across blobs and queues.

Reply:
Fixed